jsonrpc-types/src/request/tests/params.rs

```diff
@@ -227,4 +227,6 @@ fn serialize_and_deserialize() {
         ["earliest"],
         (BlockNumber::earliest())
     );
+
+    test_ser_and_de!(GetVersionParams, [], ());
```

Comment:
What's the test used for?

Reply:
just test serialization, like log.rs in the same dir.

Reply:
All params have this test, like test_ser_and_de!(BlockNumberParams, [], ()).